autonomy/cli/helpers/deployment.py

```diff
@@ -144,10 +142,9 @@ def build_deployment(  # pylint: disable=too-many-arguments, too-many-locals
     image_author: Optional[str] = None,
 ) -> None:
     """Build deployment."""
+
     if build_dir.is_dir():  # pragma: no cover
-        if not force_overwrite:
-            raise click.ClickException(f"Build already exists @ {build_dir}")
-        shutil.rmtree(build_dir)
```

Comment:
I think we should just leave this to the users since this a very trivial thing to do. Keeping this piece of code here will require some platform and user group specific hacks.